### PR TITLE
feat: data apps docs - BETA

### PR DESCRIPTION
**Pull request by MiroCillik**
Jira issue(s): [PROOF-XXX](https://keboola.atlassian.net/browse/PROOF-66)

Changes:
- Added documentation for Data Apps. It's still quite brief and rough as it is BETA

### CODE REVIEW

File: components/data-apps/index.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+There are two ways how to create a Data App in Keboola. Choose deployment type which suits your needs:
+- **Code** - Just paste a Streamlit code to create a simple Data App. 
+- **Git repository** - Specify git repository with Streamlit app sources. More suitable for complex applications.
+
+{: .image-popup}
+![Code - main menu](/components/data-apps/data apps - main menu.png)
+
+### Code
+For a simple use-cases, where your Streamlit code fits into a one page, paste the code directly into a textarea. 
+Ideal for very simple apps or testing things. Try it immediately with [this example from Streamlit docs](- https://docs.streamlit.io/library/get-started/create-an-app#lets-put-it-all-together).
```
Comment (MonikaFeigler) [inline components/data-apps/index.md:36]: can we add also link to our demo.keboola.com project please? It's also from a code:
https://demo.keboola.com/app/data-apps/45663441

File: components/data-apps/index.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+#### Packages
+To use additional packages, which are not already in our [Streamlit Base Image](#Base Image), enter it into the Packages field.
+
+{: .image-popup}
+![Packages](/components/data-apps/data apps - packages.png)
+
+### Git repository
+{% include warning.html content="In BETA we only support Github repositories." %}
+
+Please provide feedback using the feedback button in your project." %}
```
Comment (MonikaFeigler) [inline components/data-apps/index.md:50]: why is this here? Please provide feedback using the feedback button in your project." %}

File: components/data-apps/index.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+If you have a complex application, you can push your app sources into Github and link the repository in this section.
+Provide Project URL, choose the right branch and finally select your main entrypoint file.
```
Comment (MonikaFeigler) [inline components/data-apps/index.md:52]: can we add an info that currently we support only repos with MAIN branch not MASTER, please?

File: components/data-apps/index.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+## Secrets
+To provide your app with environment variables or sensitive information like credentials, api keys etc, enter them as key value pairs in the Secrets section.
+These secrets will be injected into the secrets.toml file upon deployment of the app. 
+[Read more about the Streamlit secrets](https://docs.streamlit.io/streamlit-community-cloud/get-started/deploy-an-app/connect-to-data-sources/secrets-management).
+
+{: .image-popup}
+![Secrets](/components/data-apps/data apps - secrets.png)
+
+## Data loading
+To load Data into the app, you can either use [Input mapping](https://help.keboola.com/transformations/mappings/#input-mapping) section, which will load data from Keboola project Storage into the app.
```
Comment (MonikaFeigler) [inline components/data-apps/index.md:66]: can we write here simple sentence like: Just select your table into input mapping section and navigate to that by /data/in/table/your_data.csv or /data/in/files/fileID_FileName.* in your code. That's what people need to know.

File: components/data-apps/index.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+## Secrets
+To provide your app with environment variables or sensitive information like credentials, api keys etc, enter them as key value pairs in the Secrets section.
+These secrets will be injected into the secrets.toml file upon deployment of the app. 
+[Read more about the Streamlit secrets](https://docs.streamlit.io/streamlit-community-cloud/get-started/deploy-an-app/connect-to-data-sources/secrets-management).
+
+{: .image-popup}
+![Secrets](/components/data-apps/data apps - secrets.png)
+
+## Data loading
```
Comment (MonikaFeigler) [inline components/data-apps/index.md:65]: This is data loading can we add a section about getting data back to keboola, please?

File: components/data-apps/index.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+To load Data into the app, you can either use [Input mapping](https://help.keboola.com/transformations/mappings/#input-mapping) section, which will load data from Keboola project Storage into the app.
+(Note that, while in BETA, the app needs to be redeployed to fetch up-to-date data.)
+Or you can use [Keboola Storage Python Client](https://github.com/keboola/sapi-python-client) in the app to load the data as needed.
+See [Examples](#Examples) below for usage of the Keboola Storage Python Client.
+
+## Deployment and App management
+
+{% include warning.html content="Once the Data App is deployed it's url will be publicly available! Keboola does not provide authorization to Data Apps out of the box yet." %}
```
Comment (MonikaFeigler) [inline components/data-apps/index.md:73]: Please. add a sentence that we recommend the authorization if they use their data for some vizualization or actions. They can implement simple authorization in their own code as it is in this example https://github.com/KB-PS/mkt-bi-ocr.

File: components/data-apps/index.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+### Debugging App deployment
+If the Data App's deployment job fails, you can see the logs from its container in the events log of the deployment job.  
+For example a conflict of the specified packages:
+
+{: .image-popup}
+![Secrets](/components/data-apps/data apps - job error log.png)
+
+## Example Data Apps
+
+### Customer Segmentation
```
Comment (MonikaFeigler) [inline components/data-apps/index.md:96]: Please add also this one: https://demo.keboola.com/app/data-apps/45663441
thanks.

File: components/data-apps/index.md (hunk 2 above)
Comment (MiroCillik) [reply components/data-apps/index.md:50]: I've seen it somewhere else :) I will remove it

File: components/data-apps/index.md (hunk 4 above)
Comment (MiroCillik) [reply components/data-apps/index.md:66]: OK, added

File: components/data-apps/index.md (hunk 7 above)
Comment (MiroCillik) [reply components/data-apps/index.md:96]: Jasne, sorry, ze som to tam nedal hned

**Review by MonikaFeigler — APPROVED** [review]
Content is good to go for me.

**Review by hhanova — APPROVED** [review]
ten jeden link jsem opravila, takze odkazy jsou vsechny ok

zbyvaji jen screenshoty, ktere se mi v textu pri VIEW nezobrazuji (to se mi deje pokazde, kdyz necham mezery v nazvech :)
